dipper/models/Model.py

```diff
@@ -218,7 +218,10 @@ def addOntologyDeclaration(self, ontology_id):
             ontology_id, self.globaltt['type'], self.globaltt['ontology'])
 
     def addOWLVersionIRI(self, ontology_id, version_iri):
-        self.graph.addTriple(ontology_id, self.globaltt['version_iri'], version_iri)
+        self.graph.addTriple(ontology_id,
+                             self.globaltt['version_iri'],
+                             version_iri,
+                             object_is_literal=False)
```

Comment:
@TomConlin interested in your opinion here. I'm having to add this to force the dataset version IRI (e.g. `MonarchArchive:/201908/#OMIM`) to be considered an IRI and not a literal. This change means that any ingest that sends a OWL version IRI that isn't a curie will fail with an `AssertionError` when Dipper goes to actually add the triple. That is okay I think, I'm just curious where you suggest we draw the line about when to make an ingest fail versus emit a warning and recover.

Reply:
I have done the same but within various helper functions that are intended to only emit non literal objects. What do you mean it must be a curie?  Seems sending a full iri should work as well as a curie at not being a literal.

Reply:
> What do you mean it must be a curie? Seems sending a full iri should work as well as a curie at not being a literal.

Yep, should've said CURIE or IRI

> I would make it fail if version_iri could not be a proper IRI or a curie.

Okay, good, thanks. That's the behavior here